alacritty_terminal/src/config/visual_bell.rs

```diff
@@ -43,15 +43,17 @@ impl VisualBellConfig {
 /// Penner's Easing Functions.
 #[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq)]
 pub enum VisualBellAnimation {
-    Ease,         // CSS
-    EaseOut,      // CSS
-    EaseOutSine,  // Penner
-    EaseOutQuad,  // Penner
-    EaseOutCubic, // Penner
-    EaseOutQuart, // Penner
-    EaseOutQuint, // Penner
-    EaseOutExpo,  // Penner
-    EaseOutCirc,  // Penner
+    // CSS.
+    Ease,
+    EaseOut,
+    // Penner.
+    EaseOutSine,
+    EaseOutQuad,
+    EaseOutCubic,
+    EaseOutQuart,
+    EaseOutQuint,
+    EaseOutExpo,
+    EaseOutCirc,
```

Comment:
This change doesn't correctly show that the comments affect all elements after the comment.